HW1/P5/P5_bfs.py

```diff
@@ -0,0 +1,158 @@
+import time
+import sys
+
+import pyspark
+sc = pyspark.SparkContext(appName="spark1")
+
+partition_size = 20
+default_distance = sys.maxint
+sum_distance = sc.accumulator(0)
+
+
+def copartitioned(RDD1, RDD2):
+    "check if two RDDs are copartitioned"
+    return RDD1.partitioner == RDD2.partitioner
```

Comment:
maybe instead of writing this code multiple times you could create a shared file in a parent directory or link to a file from a neighbor directory?